tests/unittest/test_bin.py

```diff
@@ -11,7 +11,7 @@
 
 
 class TestS3Cli:
-    def test_runs(self):
+    def test_basic(self):
```

Comment:
I changed the names of these tests to `test_basic` figuring that's a better name.